docs/manual/java/guide/broker/KafkaClient.md

```diff
@@ -31,7 +31,7 @@ The Lagom Kafka Client implementation is built using [akka-stream-kafka](https:/
 
 @[kafka-broker](../../../../../service/core/kafka/client/src/main/resources/reference.conf)
 
-First, notice that the `service-name` is set to "kafka_native" by default. This property defines how the kafka broker URL will be looked up in the service locator (since v1.3.1). If you choose you can disable the lookup by setting the service-name to an empty string and pass the location of your Kafka brokers via the key `lagom.broker.kafka.brokers`. In production, you will usually want to have at least two brokers for resiliency. Make sure to separate each broker URL with a comma. Note again that since v1.3.1 this property is only used if `service-name` is empty.
+First, notice that the `service-name` is set to "kafka_native" by default. This property defines how the kafka broker URI will be looked up in the service locator (since v1.3.1). If you choose you can disable the lookup by setting the service-name to an empty string and pass the location of your Kafka brokers via the key `lagom.broker.kafka.brokers`. This setting is mapped to Kafka's [boot-strap server](https://kafka.apache.org/documentation/#producerconfigs) list so only a few of the brokers need to be specified since the rest will be discovered dynamically. Make sure to separate each broker URL with a comma.
```

Comment:
Why remove this? "In production, you will usually want to have at least two brokers for resiliency." Sounds like good advice.